Gemfile.lock

```diff
@@ -6,25 +6,19 @@ GIT
     commandrb (0.4.8)
       discordrb (~> 3.1, >= 3.1.0)
 
-GIT
-  remote: https://github.com/resque/redis-namespace.git
-  revision: c31e63dc3cd5e59ef5ea394d4d46ac60d1e6f82e
-  ref: c31e63dc3cd5e59ef5ea394d4d46ac60d1e6f82e
-  specs:
-    redis-namespace (1.8.0)
-      redis (>= 3.0.4)
-
 GIT
   remote: https://github.com/shardlab/discordrb.git
-  revision: e17efe773d6281a22ead831ce7586d98dcdc32af
+  revision: 75d3e6a8fd2cda78a1e3c8e4585b33372b5800e4
   branch: main
   specs:
-    discordrb (3.4.0)
-      discordrb-webhooks (~> 3.3.0)
+    discordrb (3.4.2)
+      discordrb-webhooks (~> 3.4.2)
```

Comment:
discordrb 3.4.2/3.4.1 were pulled off RubyGems a few days ago ([relevant message within #ruby_discordrb in Discord API](https://canary.discord.com/channels/81384788765712384/381891448884428801/812489337529696336)) due to quite a few issues involving how they handle intents in larger servers, such as Discord Bots. I wanted to update a little while ago but ran into this limitation locally.

Could we pin discordrb to 3.4.0 for right now (as their GitHub repository has these versions tagged)?
Alternatively, we might want to consider using RubyGems once again, as discordrb is no longer horribly outdated with new features xP

Reply:
Sure if the maintainers are keeping things fresh now I don't see a problem with going back to RubyGems, certainly better than pinning a version from git.

Reply:
Done! https://github.com/Yuuki-Discord/Yuuki-Bot/pull/81/commits/541579941e94a9d6a12c2abbc34d5b4293e7cc14